src/vm/prestub.cpp

```diff
@@ -236,6 +236,37 @@ void DACNotifyCompilationFinished(MethodDesc *methodDesc)
     }
 }
 
+void DACNotifyCompilationFinished2(MethodDesc *methodDesc, PCODE pCode)
```

Comment:
DACNotifyCompilationFinished above is dead code I think? I'd delete it and rename this one to not have a '2'.